File: PYTHON/Daily Flash/Week2/Day1_Jan20/code/prog4.py
'''
Program 4 : Write a Progr am to accept 10 integers from user and prin t s the
sum & average of entered numbers .
Input: 1 2 3 4 5 6 7 8 9 10
Output:
Sum of 10 entered Number is: 55
Average of 10 entered number is: 5.55
'''


sum = 0;
print("Enter 10 Integers: ")
for x in range(10):
        num = int(input("Enter num: "))
        sum +=x+1;      #add

avg = sum /10
print("The sum of number up to 10: "+str(sum))
print("The average of number up to 10: "+str(avg))

File: C/Daily Flash/Week1/Day6_Jan18/code_C/prog4.c
/*
*Program 4 : Write a Program to print First 50 Even Numbers
Output: 2 4 6 . . . 100
*
*/

#include <stdio.h>

void main() {

	printf("First 50 EVEN Numbers:\n");
	int num =0;

	for(int i=1; i<= 50; i++){
		printf("%d ",num);
		num +=2;
	}
	printf("\n");
}
